ducktape/command_line/defaults.py

```diff
@@ -15,7 +15,7 @@
 import os
 
 
-class ConsoleConfig(object):
+class ConsoleDefaults(object):
```

Comment:
Not necessary, just wasn't clear to me we considered it internal. This is one of the problems w/ Python code -- it's not always obvious what is internal.